_includes/breadcrumbs.html

```diff
@@ -1,5 +1,5 @@
 {% if page.url != "/" %}
-  <nav id="breadcrumb" class="breadcrumb">
+  <nav id="breadcrumb" class="breadcrumb" aria-label="breadcrumbs">
```

Comment:
I'm not sure how common the terminology "breadcrumbs" is among non-developers, but in any case, the fact that this is a navigation element should already provide enough contents for users of screen readers.

```suggestion
  <nav id="breadcrumb" class="breadcrumb">
```